KenticoCloud.ContentManagement.Helpers.Tests/EditLinkBuilderTests.cs

```diff
@@ -97,6 +89,64 @@ public void EditLinkBuilder_ProjectIdBadFormat_ThrowsException()
             };
 
             Assert.Throws<ArgumentException>(() => new EditLinkBuilder(badFormatProjectIdOptions));
-        }   
+        }
+
+        [Fact]
+        public void BuildEditItemUrl_SingleElement_ReturnsValidUrl()
+        {
+            var elementIdentifier = new ElementIdentifier(_itemId, "4b670e0d-9e66-45f7-8a76-057022a3e47c");
+            
+            var underTest = new EditLinkBuilder(DefaultOptions);
+            var result = underTest.BuildEditItemUrl(_language, elementIdentifier);
+
+            var expected = "https://app.kenticocloud.com/goto/edit-item/project/14dc0cf8-6cc1-4f20-8e2e-0b5edea89e43/variant-codename/some-Language-Codename/item/1cdaa8ef-cb2b-4f82-82e9-45467b2e01b9/element/4b670e0d-9e66-45f7-8a76-057022a3e47c";
+            Assert.Equal(expected, result);
+        }
+
+        [Fact]
+        public void BuildEditItemUrl_MultipleElements_ReturnsValidUrl()
+        {
+            var elements = new ElementIdentifier[]
+            {
+                new ElementIdentifier("76c06b74-bae9-4732-b629-1a59395e893d", "4b670e0d-9e66-45f7-8a76-057022a3e47c"),
+                new ElementIdentifier("326c63aa-ae71-40b7-a6a8-56455b0b9751", "0369432d-d7e1-411f-af1a-b318bfd1b38e"),
+                new ElementIdentifier("ffcd0436-8274-40ee-aaae-86fee1966fce", "201ea928-5866-4a9c-aac2-e8bf41a5bc52"),
+                new ElementIdentifier("d31d27cf-ddf6-4040-ab67-2f70edc0d46b", "ace3f21b-92d3-4834-b50a-ce2b308eac86"),
```

Comment:
It would be better to have element code names as names, not GUIDs, as tests serve also as a kind of documentation